packages/rum/src/domain/segmentCollection/startDeflateWorker.spec.ts

```diff
@@ -142,7 +142,12 @@ describe('startDeflateWorker', () => {
         throw UNKNOWN_ERROR
       })
       expect(telemetryEvents).toEqual([
-        { status: 'error' as any, message: 'boom', error: { kind: 'Error', stack: jasmine.any(String) } },
+        {
+          type: 'log' as any,
+          status: 'error' as any,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Could we get rid of those `any`?